src/osd/scheduler/mClockScheduler.h

```diff
@@ -130,7 +131,12 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
     true,
     true,
     2>;
+  using SubQueue = std::map<unsigned,
```

Comment:
`using priority_t = unsigned;` and substituting in the declaration would make the purpose of the SubQueue much clearer.